### ISSUE TITLE
Video Controls Focus Indicator

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6054]
### Expected Behavior
When navigating through the video controls with the keyboard the focus indicator should be visible.

### Actual Behavior
The focus indicator is not visible

### URL, screen shot, or Codepen exhibiting the issue
This can be seen in the Video/Theater mode story https://storybook.grommet.io/?path=/story/media-video-theater-mode--customizable-controls

### Steps to Reproduce

1. Go to https://storybook.grommet.io/?path=/story/media-video-theater-mode--customizable-controls
2. Navigate to the controls menu with the keyboard and open it
3. Navigate within the controls menu with the keyboard and the focus indicator is not visible

### Your Environment

- Grommet version: 2.22.0
- Browser Name and version: chrome
- Operating System and version (desktop or mobile):

**Comment by Ojas-Sri** [comment]
This looks like an issue with the `Menu` Component. Seems like the video controls thing uses a `Menu` component.

 For example in this `<Menu/>` [storybook](https://storybook.grommet.io/?path=/story/controls-menu-reverse--reverse) . I am not getting focus on Menu items, on navigating via Keyboard.

**Comment by jcfilben** [comment]
Yeah I think it is related to Menu, we should get the focusIndicator on menu when navigating with the keyboard.

**Comment by jeferson-sb** [comment]
I think this would be pretty much just setting `focusIndicator`  on menu items to true

https://github.com/grommet/grommet/blob/1edfd5166a2d61785e4eb52a8ae539c1c0619360/src/js/components/Menu/Menu.js#L351-L363

**Comment by jcfilben** [comment]
> I think this would be pretty much just setting focusIndicator on menu items to true

Yes, the key is we only want to show the focusIndicator when navigating with the keyboard
